pepper-apis/config/squid-docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+squid:
+  image: datadog/squid:latest
```

Comment:
There are other images from random people on github. I figured using an image from a legitimate tech company will be better. If you have a favorite image we should use, let me know!

Reply:
I don't have one- my last squid installs were in the early days of Docker. I'm also a bit wary that the `Dockerfile` for that image is not available.

The readme references [sameersbn/docker-squid](https://github.com/sameersbn/docker-squid) as the source, and the `Dockerfile` doesn't looks terribly complicated. I'd recommend just creating a local `Dockerfile` instead of pulling in an external dependency for this, or using `sameersbn`'s directly (with proper attribution & respect to the MIT license it's under)

Reply:
I couldn't find a github repo for datadog's squid image either. But looks like you can browse the image layers on dockerhub: https://hub.docker.com/layers/datadog/squid/latest/images/sha256-f7d19d5e3f4163771291d91de393ce667f2327a3e080c39b9b7ea9e19f91488f?context=explore.

Reply:
Yeah. We should do our own Dockerfile.

Reply:
"our own dockerfile" as in source it from datadog, or source it from sameersbn, or do it from scratch?

Reply:
We can't use Datadog's since they don't have it available, so it's either be write one up from scratch to build the image, or use `sameersbn` to create an image.

Reply:
Done.